lib/todo.rb

```diff
@@ -15,6 +15,10 @@ def database
       @config[:database]
     end #database
 
+    def contents
+      @contents = get_contents
```

Comment:
Needs memoization here.
```suggestion
      @contents ||= get_contents
```

Reply:
Fixed